addons/account/i18n/account.pot

```diff
@@ -4464,11 +4479,6 @@ msgid "Invoices to validate"
 msgstr ""
 
 #. module: account
-#: model:ir.ui.menu,name:account.menu_finance
-msgid "Invoicing"
```

Comment:
Ok so this is an example when you should **not** use db-all :-) [Invoicing](https://github.com/odoo/odoo/blob/6327cda0a79a6cbae1d5f9981aed27d26cdad4fe/addons/account/views/account_menuitem.xml#L6) and [Accounting](https://github.com/odoo/odoo/blob/6327cda0a79a6cbae1d5f9981aed27d26cdad4fe/addons/account_accountant/account_accountant_data.xml#L6)